### ISSUE TITLE
caller_info broken after upgrade to 1.0

### ISSUE THREAD

**Issue by grr** [garu/Data-Printer#156]
.dataprinter consists of only a single line:
```caller_info = 1```

script:
```
#!/usr/bin/env perl
use strict;
use warnings;
use 5.012;
use DDP;

my $h = { a => 1, 2 => 'b'};
p $h;
```

ouput:
```
Use of uninitialized value $caller[1] in substitution iterator at /perlbrew/perls/perl-5.32/lib/site_perl/5.32.0/Data/Printer/Object.pm line 761.
Use of uninitialized value $caller[2] in substitution iterator at /perlbrew/perls/perl-5.32/lib/site_perl/5.32.0/Data/Printer/Object.pm line 762.
Printing in line  of :
{
    2   "b",
    a   1
}
```

system info:
```
$ perl -E 'say $]'
5.032000
$ perl -E 'say $^O'
darwin
```

**Comment by garu** [comment]
Thank you for the report! I was able to reproduce and will push a patched version with regression tests later tonight!

FYI, it's an issue with converting the `__FILENAME__` and `__LINE__` special arguments on the default caller string. The warnings go away if you use anything else, e.g.:

```
caller_info = 1
caller_message = "Data::Printer output:"
```

I'll update this issue as soon as the patched version is released!

**Comment by garu** [comment]
Fixed in 1.0.1 🥳 

Let me know if you bump into any issues.

Cheers!

**Comment by grr** [comment]
Unfortunately, the new default theme 'Material' renders the caller_info string in the color 'very_dark_gray', which is illegible on a terminal with a dark background. Works fine in all other themes.
